libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -62,6 +61,16 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
    */
   @Input() trackBy: TrackByFunction<any>;
 
+  /**
+   * Property name to decide which items should be stand alone
+   */
+  @Input() getStandAloneByProperty: string;
+
+  /**
+   * Bottom margin for stand alone items
+   */
+  @Input() standAloneMargin: string = '8px';
```

Comment:
I see several problems with this approach: 

1. Default margin is _hardcoded_ as a _magic number_ in a _string_ in _Typescript_ (3-4 red flags for the price of 1 😉). I would expect margin to be declared in a stylesheet along with the remaining styles
2. We're exposing this as an option where consumers can easily do something unintended, like assigning an arbitrary margin value that doesn't adhere to the design guidelines.
3. We're using pixels as unit. While this is not incorrect it's usually better to use units relative to the users' font size, such as em, rem, ch and ex for a bunch of reasons. Our design tokens _are_ unfortunately defined using pixel units (so this only counts as half a red flag 😁)

Other approaches to consider:

- Use a CSS custom property; move styles to a stylesheet. This will give consumers the same option, but it will also have the same drawbacks as mentioned above (especially pt. 2. and 3.)
- Use Kirby's size design tokens. This will allow consumers to choose between predefined "correct values". They cannot choose whatever margin they want, but it will ensure that it adheres to the design decisions. 

Another solution to consider is to not apply margin at all. One could argue that whitespace between lists or list sections is part of external layout and should be applied by the consumer.

Maybe combining multiple approaches will result in an ideal solution. I'll be happy to discuss possible solutions in detail if you're interested 🙂